package.json

```diff
@@ -33,8 +33,8 @@
     "next-plausible": "^3.11.1",
     "next-sitemap": "^4.2.3",
     "node-ical": "0.16.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "^18.2.0",
```

Comment:
Why this change?

Reply:
@VivekJaiswal18 Why this change?

Reply:
@benjagm  I introduced this change so that if minor patches were introduced by the library then anyone installing it can benefit from it (if anyone does npm install react/react-dom). 
But if you think it should not be done then I can remove it.

Reply:
I think its good to remove that change since it also modifies files like `package-lock.json` and `yarn.lock` and ultimately increases the PR size.

![image](https://github.com/json-schema-org/website/assets/124715224/c23f90eb-bafe-4b0f-b56d-af69e0e1b3ea)